server/api/attendance/index.js

```diff
@@ -41,4 +41,7 @@ router.get('/unverified/:date',  auth.hasRole('mentor'), controller.getUnverifie
 router.get('/unverified/:date/full', auth.hasRole('mentor'), controller.getUnverifiedFullAttendanceUsers);
 router.get('/unverified/:date/small', auth.hasRole('mentor'), controller.getUnverifiedSmallAttendanceUsers);
 
+//Gets the number of attendees on a date
+router.get('/code/:dateCode',auth.hasRole('mentor'), controller.getAttendees);
```

Comment:
It would be useful to have the endpoint and controller name reflect that it is the number of attendees. Another endpoint to provide all the attendees would be nice to have as a future feature.